src/applications/user-testing/disability-benefits-all-claims-conditions-chapter/config/form.js

```diff
@@ -52,7 +51,6 @@ const formConfig = {
     conditionsChapter: {
       title: 'Conditions',
       pages: {
-        demo,
         ...ratedOrNewNextPagePages,
         ...conditionTypeRadioPages,
         ...ratedOrNewNextPageSecondaryEnhancedPages,
```

Comment:
It is not necessary to remove the ...conditionTypeRadioPages and ...conditionTypeRadioPages as the application never sees this workflow. However, keeping them here facilitates re-introducing the kiwi/papaya workflow.